app/src/main/res/layout/item_places_list.xml

```diff
@@ -9,50 +9,49 @@
     android:paddingVertical="12dp"
     android:paddingHorizontal="@dimen/list_item_horizontal_padding">
 
-    <TextView
-        android:id="@+id/listItemTitle"
-        style="@style/H3.Article"
+    <LinearLayout
```

Comment:
I use `LinearLayout` to wrap up items in one container so it can align to the center correctly if no descriptions or has multiple lines on article title.